examples/grpc/greeter/client.cpp

```diff
@@ -0,0 +1,84 @@
+#include "greeter.grpc.fb.h"
```

Comment:
Thanks for adding another example!
Not sure about the top level `examples` dir, since we already have `samples` we could put it under, or better yet use `grpc/samples` since we already are keeping everything grpc related under there, including tests.